index.bs

```diff
@@ -496,33 +503,113 @@ Note: {{XRReferenceSpaceType/"local"}} is always included in the [=requested fea
 
 Note: For example, several VR devices support either configuring a safe boundary for the user to move around within or skipping boundary configuration and operating in a mode where the user is expected to stand in place. Such a device can be considered to be [=capable of supporting=] {{"bounded-floor"}} {{XRReferenceSpace}}s even if they are currently not configured with safety boundaries, because it's expected that the user could configure the device appropriately if the experience required it. This is to allow user agents to avoid fully initializing the [=XRSession/XR device=] or waiting for the user's environment to be recognized prior to [=resolve the requested features|resolving the requested features=] if desired. If, however, the user agent knows that the boundary state at the time the session is requested without additional initialization it may choose to reject the {{"bounded-floor"}} feature if the safety boundary not already configured.
 
+Permissions API Integration {#permissions}
+---------
```

Comment:
I understand why this was put here, but it feels currently like it's kind of "breaking the flow" of the document. (As much as a spec can have any sort of proper flow to it, anyway.) It's not really part of the "Initialization" topic, nor is it a critical piece to using the API. As a result, despite the strong relationship to the `requiredFeatures` and `optionalFeatures`, I feel this would be better placed near the end of the spec (in the "Integrations" section).

Reply:
Works for me.